ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/Examples.hs

```diff
@@ -143,6 +149,30 @@ instance Inject Examples where
         => Proxy f -> Labelled a -> Labelled b
       inj p = fmap (fmap (inject' p startBounds idx))
 
+newtype WrapLedgerTables blk = WrapLedgerTables ( LedgerTables (ExtLedgerState blk) ValuesMK )
```

Comment:
If it stays, let's at least comment explaining the narrow use case of this newtype.